aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,51 @@
+import { Example } from '../../layouts';
+import { PageExample, PageContentExample } from '../../examples';
+
+**Page** is the main container that is responsible for how the Page Content and Page Header are 
+presented. 
```

Comment:
Our introductory content on a page should really try to set up the "what" AND "why". In this case, the "why" is solving the problem of "how do we make it easy for all page layouts within an application and across all HPE applications to be implemented consistently?"

```suggestion
Page makes it easy to implement [page layouts](/templates/page-layouts) uniformly across HPE applications. Two components, Page and PageContent, work together to ensure HPE theme-defined margins, horizontal alignment, max-width, and responsive behaviors are applied in a consistent, turn-key manner.
```